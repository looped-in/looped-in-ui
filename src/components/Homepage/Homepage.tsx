import Footer from "@/components/Footer";
import Header from "@/components/Header";
import TopBannerHomepage from "@/components/TopBannerHomepage";
import React from "react";
import FindBuilder from "../FindBuilder";
import Processes from "../Processes";

const Homepage = () => {
	return (
		<div>
			<Header />

			<TopBannerHomepage />

			<FindBuilder />

			<Processes />

			<Footer />
		</div>
	);
};

export default Homepage;
