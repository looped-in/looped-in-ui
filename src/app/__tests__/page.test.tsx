import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Homepage from "@/components/Homepage";

describe("Page", () => {
	it("SHOULD match snapshot", () => {
		const { container } = render(<Homepage />);
		expect(container).toMatchSnapshot();
	});

	it("SHOULD renders correct heading", () => {
		render(<Homepage />);
		expect(
			screen.getByText(
				"Collaboration tools you'll love for the jobs you don't.",
			),
		).toBeInTheDocument();
	});
});
