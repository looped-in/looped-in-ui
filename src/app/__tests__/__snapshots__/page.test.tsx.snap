// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page SHOULD match snapshot 1`] = `
<div>
  <div>
    <div
      class="p-16"
    >
      <div
        class="flex flex-row place-content-between"
      >
        <img
          alt="logo"
          data-nimg="1"
          decoding="async"
          height="150"
          loading="lazy"
          src="/logo.svg"
          style="color: transparent;"
          width="150"
        />
        <div
          class="flex flex-row space-x-6 align-middle content-around text-sm font-semibold"
        >
          <a
            class="content-around hover:text-[#ff7338]"
            href="/"
          >
            About
          </a>
          <a
            class="content-around hover:text-[#ff7338]"
            href="/how-it-work"
          >
            How it works
          </a>
          <a
            class="content-around hover:text-[#ff7338]"
            href="/our-team"
          >
            Our Team
          </a>
          <a
            class="content-around hover:text-[#ff7338]"
            href="/find-builder"
          >
            Find a builder
          </a>
          <a
            class="content-around hover:text-[#ff7338]"
            href="/get-in-touch"
          >
            Get in touch
          </a>
          <div
            class="content-around align-middle hover:text-[#ff7338]"
          >
            <a
              href="/sign-in"
            >
              <button
                class="h-10 w-32 rounded-tl-lg rounded-tr-lg rounded-br-lg bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-white font-bold"
              >
                Sign in
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="flex flex-row p-10 space-x-4"
    >
      <div
        class="w-1/2 space-y-6"
      >
        <div
          class="text-4xl font-medium"
        >
          Collaboration tools you'll love for the jobs you don't.
        </div>
        <div
          class="text-sm"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Luctus morbi est porttitor velit, sit bibendum vitae blandit. Viverra orci ornare vehicula mauris purus pulvinar diam, vel. Odio pharetra mauris metus scelerisque.
        </div>
        <div
          class="text-sm flex flex-row "
        >
          <a
            class="w-1/3"
            href="/sign-in"
          >
            <button
              class="h-10 w-32 rounded-tl-lg rounded-tr-lg rounded-br-lg bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-white font-bold"
            >
              Sign in
            </button>
          </a>
          <div
            class="text-center w-1/2 content-around"
          >
            Find a Builder
          </div>
        </div>
      </div>
      <div
        class="w-1/2"
      >
        <img
          alt="homepage-banner"
          class="rounded-xl"
          data-nimg="1"
          decoding="async"
          height="1000"
          loading="lazy"
          src="/_next/image?url=%2Ffind-builder.png&w=2048&q=75"
          srcset="/_next/image?url=%2Ffind-builder.png&w=1080&q=75 1x, /_next/image?url=%2Ffind-builder.png&w=2048&q=75 2x"
          style="color: transparent;"
          width="1000"
        />
      </div>
    </div>
    <div
      class=" flex flex-row h-3/6"
    >
      <div
        class="w-3/6"
      >
        <img
          alt="find-builder"
          data-nimg="1"
          decoding="async"
          height="1500"
          loading="lazy"
          src="/_next/image?url=%2Ffind-builder.png&w=3840&q=75"
          srcset="/_next/image?url=%2Ffind-builder.png&w=1920&q=75 1x, /_next/image?url=%2Ffind-builder.png&w=3840&q=75 2x"
          style="color: transparent;"
          width="1500"
        />
      </div>
      <div
        class="w-3/6 bg-zinc-800 content-around p-8 space-y-6"
      >
        <div
          class="text-transparent bg-clip-text bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-4xl font-semibold"
        >
          Find a builder for 
          <br />
          your next job!
        </div>
        <div
          class="text-slate-50 text-sm "
        >
          Building or renovating? Let Loopedin help you find the right builder for your next big project. Browse to find your builder!
        </div>
        <button
          class="h-10 rounded-tl-lg rounded-tr-lg rounded-br-lg bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-white font-bold w-1/2"
        >
          Find a Builder
        </button>
      </div>
    </div>
    <div
      class="flex flex-col py-20"
    >
      <div
        class="text-center text-6xl font-semibold"
      >
        Loopedin the right way!
      </div>
      <div
        class="flex flex-row py-50"
      >
        <div
          class="w-1/2 space-y-6 p-4 place-content-center"
        >
          <button
            class="h-10 w-32 rounded-tl-lg rounded-tr-lg rounded-br-lg bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-white font-bold"
          >
            Step 1
          </button>
          <div
            class="text-2xl font-semibold"
          >
            Step One of The Process
          </div>
          <div
            class="text-sm"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Luctus morbi est porttitor velit, sit bibendum vitae blandit. Viverra orci ornare vehicula mauris purus pulvinar diam, vel. Odio pharetra mauris metus scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Luctus morbi est porttitor velit, sit bibendum vitae blandit. Viverra orci ornare vehicula mauris purus pulvinar diam, vel. Odio pharetra mauris metus scelerisque.
          </div>
        </div>
        <div
          class="w-1/2"
        >
          <img
            alt="step-1"
            data-nimg="1"
            decoding="async"
            height="1000"
            loading="lazy"
            src="/_next/image?url=%2Fstep-one.png&w=2048&q=75"
            srcset="/_next/image?url=%2Fstep-one.png&w=1080&q=75 1x, /_next/image?url=%2Fstep-one.png&w=2048&q=75 2x"
            style="color: transparent;"
            width="1000"
          />
        </div>
      </div>
      <div
        class="flex flex-row py-2"
      >
        <div
          class="w-1/2"
        >
          <img
            alt="step-2"
            data-nimg="1"
            decoding="async"
            height="1000"
            loading="lazy"
            src="/_next/image?url=%2Fstep-2.1.png&w=2048&q=75"
            srcset="/_next/image?url=%2Fstep-2.1.png&w=1080&q=75 1x, /_next/image?url=%2Fstep-2.1.png&w=2048&q=75 2x"
            style="color: transparent;"
            width="1000"
          />
        </div>
        <div
          class="w-1/2 space-y-6 p-4 place-content-center"
        >
          <button
            class="h-10 w-32 rounded-tl-lg rounded-tr-lg rounded-br-lg bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-white font-bold"
          >
            Step 2
          </button>
          <div
            class="text-2xl font-semibold"
          >
            Step Two of The Process
          </div>
          <div
            class="text-sm"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Luctus morbi est porttitor velit, sit bibendum vitae blandit. Viverra orci ornare vehicula mauris purus pulvinar diam, vel. Odio pharetra mauris metus scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Luctus morbi est porttitor velit, sit bibendum vitae blandit. Viverra orci ornare vehicula mauris purus pulvinar diam, vel. Odio pharetra mauris metus scelerisque.
          </div>
        </div>
      </div>
      <div
        class="flex flex-row py-5 "
      >
        <div
          class="w-1/2 space-y-6 p-8 place-content-center"
        >
          <button
            class="h-10 w-32 rounded-tl-lg rounded-tr-lg rounded-br-lg bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-white font-bold"
          >
            Step 1
          </button>
          <div
            class="text-2xl font-semibold"
          >
            Step One of The Process
          </div>
          <div
            class="text-sm"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Luctus morbi est porttitor velit, sit bibendum vitae blandit. Viverra orci ornare vehicula mauris purus pulvinar diam, vel. Odio pharetra mauris metus scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Luctus morbi est porttitor velit, sit bibendum vitae blandit. Viverra orci ornare vehicula mauris purus pulvinar diam, vel. Odio pharetra mauris metus scelerisque.
          </div>
        </div>
        <div
          class="w-1/2"
        >
          <img
            alt="step-1"
            data-nimg="1"
            decoding="async"
            height="1000"
            loading="lazy"
            src="/_next/image?url=%2Fstep-3.1.png&w=2048&q=75"
            srcset="/_next/image?url=%2Fstep-3.1.png&w=1080&q=75 1x, /_next/image?url=%2Fstep-3.1.png&w=2048&q=75 2x"
            style="color: transparent;"
            width="1000"
          />
        </div>
      </div>
    </div>
    <div
      class="bg-[#2E2A30] p-20"
    >
      <img
        alt="logo"
        data-nimg="1"
        decoding="async"
        height="100"
        loading="lazy"
        src="/logo-white.svg"
        style="color: transparent;"
        width="100"
      />
      <div
        class="pt-5 text-white text-4xl"
      >
        Stay in the Loop!
      </div>
      <div
        class="text-white py-2"
      >
        Subscribe by entering your email below to stay in the loop.
      </div>
      <div
        class="space-x-2 py-6 flex flex-row"
      >
        <input
          class="flex h-10 rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50 w-30"
          placeholder="Enter email"
        />
        <button
          class="h-10 w-32 rounded-tl-lg rounded-tr-lg rounded-br-lg bg-gradient-to-b from-[#ffb945] to-[#ff7338] text-white font-bold"
        >
          Subcribe
        </button>
      </div>
      <div
        class="text-white"
      >
        Find us on,
      </div>
      <div
        class="flex flex-row place-content-between text-sm pt-4"
      >
        <div
          class="flex flex-row text-white space-x-2 "
        >
          <div>
            Instagram
          </div>
          <div>
            Facebook
          </div>
          <div>
            Youtube
          </div>
          <div>
            Linkedin
          </div>
        </div>
        <div
          class="text-white flex flex-row space-x-3"
        >
          <div>
            Term & Conditions
          </div>
          <div>
            |
          </div>
          <div>
            Privacy
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
